pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java

```diff
@@ -607,9 +614,9 @@ public void skippedStagesForFailure() throws Exception {
                 .logNotContains("I will be skipped", "I also will be skipped", "I have succeeded")
                 .go();
 
-        assertTrue(b.getExecution().getCauseOfFailure() != null);
-
         FlowExecution execution = b.getExecution();
+        assertNotNull(execution);
+        assertTrue(execution.getCauseOfFailure() != null);
```

Comment:
`assertNotNull` provides better failure messages. Or `assertThat(o, not(null()))`

Reply:
durrr